tensorflow/core/kernels/BUILD

```diff
@@ -7673,6 +7673,26 @@ cc_library(
     ],
 )
 
+tf_kernel_library(
+    name = "deepspeech_cwise_ops",
```

Comment:
Is it something that should be pushed upstream ?

Reply:
Ideally, but past attempts at changing their rules to make it easier to use from outside were unsuccessful. I'm not sure they would take the change.